pkg/conversions/types_conversion.go

```diff
@@ -19,3 +19,44 @@ func Convert_CRD_Duration_To_SDK_Duration(crdObj *appmesh.Duration, sdkObj *appm
 	sdkObj.Value = aws.Int64((int64)(crdObj.Value))
 	return nil
 }
+
+func Convert_CRD_HTTPHeaderMatchMethod_To_SDK_HttpHeaderMatchMethod(crdObj *appmesh.MatchMethod, sdkObj *appmeshsdk.HeaderMatchMethod) error {
+	sdkObj.Exact = crdObj.Exact
```

Comment:
Please do add test cases where these variables are not set (i.e.,) crdObj.Exact/Prefix are nil